lib/dl_api_lib_testing/dl_api_lib_testing/connector/data_api_suites.py

```diff
@@ -472,3 +474,74 @@ def test_basic_preview(
         preview_rows = get_data_rows(preview_resp)
         min_preview_row_cnt = 10  # just an arbitrary number
         assert len(preview_rows) > min_preview_row_cnt
+
+
+class DefaultConnectorDataCacheTestSuite(StandardizedDataApiTestBase, RegulatedTestCase):
+    data_caches_enabled = True
+
+    def test_cache_with_filter_with_constants(
+        self,
+        control_api: SyncHttpDatasetApiV1,
+        data_api: SyncHttpDataApiV2,
+        db: Db,
+        saved_connection_id: str,
+    ):
+        # This used to behave incorrectly in MySQL
```

Comment:
is it relevant still?

Reply:
Well, yes - this is the reason for adding this test. Without this comment it would be a bit unclear why this test is needed